presto-hive/src/main/java/io/prestosql/plugin/hive/util/Statistics.java

```diff
@@ -99,12 +99,18 @@ public static HiveBasicStatistics reduce(HiveBasicStatistics first, HiveBasicSta
 
     public static Map<String, HiveColumnStatistics> merge(Map<String, HiveColumnStatistics> first, Map<String, HiveColumnStatistics> second)
     {
-        // only keep columns that have statistics for both sides
-        Set<String> columns = intersection(first.keySet(), second.keySet());
-        return columns.stream()
+        verify(first != null, "first is null");
```

Comment:
we usually use `requireNonNull` like here

https://github.com/prestosql/presto/blob/d36b1053d733cfd6d1c2bf043c7ca321eae93da2/presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java#L914-L915